content/articles/external-dns-providers.markdown

```diff
@@ -0,0 +1,69 @@
+---
+title: External DNS Providers
+excerpt: Link an external DNS provider to your DNSimple account to manage zones at other authoritative DNS providers, all within DNSimple.
+categories:
+- DNS
+---
+
+# External DNS Providers
+
+ DNSimple supports the ability to view and manage zones that are deployed in other DNS providers external to DNSimple. In DNSimple, we refer to such providers as "external DNS providers", and zones as ["external zones"](/articles/managing-external-zones).
+
+## Supported Features
+
+- **Import external zones**: When you link an external DNS provider to your DNSimple account, all the zones hosted on that external DNS provider will be imported into DNSimple and listed on the [Zones](/articles/managing-external-zones) page.
+- **Management of external zone records**: List, create, update and delete external zone records from DNSimple, using the [Deployment Editor](/articles/deployment-editor). (Note: The list of supported record types differ based on the provider)
+- **2-way Syncing of Records**: Sync your external zone records from the external zone to DNSimple, or from DNSimple to the external zone, with the [Deployment Editor](/articles/deployment-editor#record-syncing).
+
+## Supported External DNS Providers
+
+In order to manage [external zones](/articles/managing-external-zones) from DNSimple, you first have to link an external DNS provider to your DNSimple account.
+
+Currently, DNSimple supports the management of zones from the below authoritative DNS providers.
+
+### Amazon Route 53
+
+DNSimple supports [Amazon Route 53](https://aws.amazon.com/route53/) as an external DNS provider.
+
+#### Prerequisites for linking Amazon Route 53 as an external DNS provider
+
+To proceed with linking [Amazon Route 53](https://aws.amazon.com/route53/) as an external DNS provider to your account, you must:
+
+- Have an [AWS access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) for an AWS user with permission to manage public hosted zones at Route 53.
+- Have administrator access to a DNSimple account
+
+## Supported record types {#supported-record-types}
+
+Records of supported record types can be [synced](/articles/deployment-editor#record-syncing) from/to DNSimple to/from an external DNS provider.
+However, record fields that are not supported in DNSimple records will not be synced. Likewise, record fields like [record notes](/articles/record-notes) and [regional record](/articles/regional-records) information are only supported in DNSimple records and will not be synced to the external DNS provider.
```

Comment:
```suggestion
However, record fields that are not supported by the destination provider will be ignored. Likewise, record fields like [record notes](/articles/record-notes) and [regional record](/articles/regional-records) information are only supported by DNSimple and will not be synced to the external DNS provider.
```

Reply:
What do you think of `record fields that are not supported by DNSimple, or the destination provider` ?

Reply:
@dallasread Made the change: https://github.com/dnsimple/dnsimple-support/pull/1032/files/ec05b9231e50ff9f885b25711558094ded0be737..a775c9357ef914c11a8fb1035aa2ed1dbd69710b (cc @itsalyse)

Reply:
@dallasread Made the change: https://github.com/dnsimple/dnsimple-support/pull/1032/files/ec05b9231e50ff9f885b25711558094ded0be737..a775c9357ef914c11a8fb1035aa2ed1dbd69710b (cc @itsalyse )